src/main/java/com/zuniorteam/racingcar/view/console/ConsoleGameResultView.java

```diff
@@ -3,35 +3,47 @@
 import com.zuniorteam.racingcar.dto.GameResult;
 import com.zuniorteam.racingcar.dto.StepResult;
 import com.zuniorteam.racingcar.view.GameResultView;
+import com.zuniorteam.racingcar.vo.MoveHistory;
 
 import java.util.List;
 
 public class ConsoleGameResultView implements GameResultView {
 
     private static final String NEW_LINE = "\n";
     private static final String DASH = "-";
+    private static final String BORDER = " : ";
 
     public void draw(GameResult result) {
+        System.out.println("실행 결과");
+        System.out.println(describeGameResult(result));
+    }
 
+    private String describeGameResult(GameResult result) {
         final StringBuilder drawing = new StringBuilder();
 
         final List<StepResult> stepResults = result.getStepResults();
+
         for (StepResult stepResult : stepResults) {
             drawing.append(drawStep(stepResult))
-                    .append("\n");
+                    .append(NEW_LINE);
         }
 
-        System.out.println("실행 결과");
-        System.out.println(drawing.toString().trim());
+        drawing.append(NEW_LINE);
+        drawing.append(NEW_LINE);
 
+        drawing.append(drawWinners(result.getWinners()));
+        return drawing.toString().trim();
     }
 
     private String drawStep(StepResult stepResult) {
         final StringBuilder stepDrawing = new StringBuilder();
-        final List<Integer> carPositions = stepResult.getCarPositions();
+        final List<MoveHistory> moveHistories = stepResult.getMoveHistories();
 
-        for (Integer position : carPositions) {
-            stepDrawing.append(drawLine(position))
+        for (MoveHistory moveHistory : moveHistories) {
+            stepDrawing
```

Comment:
append 한줄정도는..붙어도 되지 않을까 싶습니다.
```suggestion
            stepDrawing.append(moveHistory.getCarName())
```